docs/docs/code/nodejs/README.md

```diff
@@ -68,37 +65,43 @@ Accepting a single string is just one example, you can build a step to accept ar
 
 [Read the props reference for the full list of options](/components/api/#props).
 
-## `async` function declaration
```

Comment:
This section still referenced the old function declaration so I modified to discuss components. lmk if you think this can be improved